packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-drilldown.vue

```diff
@@ -256,6 +266,18 @@ const newBlankInterventionPolicy = ref({
 	interventions: [blankIntervention]
 });
 
+interface SelectedIntervention {
+	index: number;
+	name: string;
+	extractionDocumentId?: string;
+	extractionPage?: number;
+	staticInterventions: StaticIntervention[];
+	dynamicInterventions: DynamicIntervention[];
+}
+
+const pdfViewer = ref();
```

Comment:
You can remove the `getPdfRef()`
```suggestion
const pdfViewer = computed(() => pdfPanelRef.value?.pdfRef[0])
```